fastlane/Fastfile

```diff
@@ -252,7 +252,8 @@ platform :ios do
       previous_text: "REVENUECAT_LOAD_SHEDDER_API_KEY",
       new_text: ENV["REVENUECAT_LOAD_SHEDDER_API_KEY"],
       paths_of_files_to_update: [
-        './Tests/BackendIntegrationTests/Constants.swift'
+        './Tests/BackendIntegrationTests/Constants.swift',
+        './Tests/v3LoadShedderIntegration/v3LoadShedderIntegrationTests/V3LoadShedderIntegrationTests.swift'
```

Comment:
Would it be worth separating it to a `Constants` file in the new project as well? Mostly to avoid replacing unintended things

Reply:
maybe? It feels a bit YAGNI - the whole file has a single constant, and these are integration tests based on a version of the SDK that's in maintenance mode, so it's not like this test suite is really going to grow over time

Reply:
also, the variable is called `REVENUECAT_LOAD_SHEDDER_API_KEY`, I don't feel like we're going to be running into a lot of conflicts with it. 
I'd vote to just keep it like this, as dead simple as possible, until the day comes where we actually start adding more to this and it becomes harder to manage.